### ISSUE TITLE
feat(app): add welcome screen 

### ISSUE THREAD

**Issue by dariadia** [dariadia/panic-room#4]
### What
Users see the welcome screen with questions about their settings.

### Why
UX && WCAG.
First, we ask users about their vision and attention para-abilities.  

**Users can select** 
- reduced motion,
- black and white theme,
- no flashing,
- etc.
